ios/FluentUI/PreviewCard/Attachment.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// UIKit wrapper that exposes the SwiftUI Attachment implementation.
+@objc open class MSFAttachment: NSObject, FluentUIWindowProvider {
+    /// The UIView representing the Attachment.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+    /// Creates a new MSFAttachment instance.
+    /// - Parameters:
+    ///   - theme: The FluentUIStyle instance representing the theme to be overriden for this Attachment.
+    ///   - thumbnail: The UIImage used as a thumbnail within the Attachment.
+    ///   - isElevated: Boolean determining if PreviewCard is flat or elevated.`
+    @objc public init(theme: FluentUIStyle?, thumbnail: UIImage, isElevated: Bool = false) {
+        super.init()
+        attachmentView = Attachment(isElevated: isElevated, thumbnail: thumbnail)
+        hostingController = FluentUIHostingController(rootView: AnyView(attachmentView.windowProvider(self)))
+        hostingController.disableSafeAreaInsets()
+        hostingController.view.backgroundColor = .clear
+    }
+    var window: UIWindow? {
+        return self.view.window
+    }
+    private var hostingController: FluentUIHostingController!
+    private var attachmentView: Attachment!
+}
+
+/// View that represents the Attachment.
+public struct Attachment: View {
+    var isElevated: Bool
+    var thumbnail: UIImage
+    @Environment(\.theme) var theme: FluentUIStyle
+    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    /// Creates Attachment's thumbnail view.
+    @ViewBuilder
+    var attachmentThumbnail: some View {
+        HStack {
+            Image(uiImage: thumbnail)
+                .frame(width: Constants.attachmentThumbnailWidth, height: Constants.attachmentThumbnailHeight)
+                .padding(EdgeInsets(top: Constants.attachmentThumbnailPaddingTop, leading: Constants.attachmentThumbnailPaddingLeading, bottom: Constants.atachmentZeroValue, trailing: Constants.atachmentZeroValue))
+        }
+    }
+    /// Creates Attachment's entire text view.
+    @ViewBuilder
+    var attachmentText: some View {
+        VStack(alignment: .leading) {
+            Text("Text")
```

Comment:
The `"Text"` and `"Subtext"` values here seem like perfect candidates to also be passed into the control at init, like the `image` value for `PreviewCard`.